CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix Exchange folder cache population error when parent folder isn't found.
 - Fix Exchange backup issue caused by incorrect json serialization
 - Fix issues with details model containing duplicate entry for api consumers
+- Handle OLE conversion errors when trying to fetch attachments
```

Comment:
So we'll skip the attachment to progress the backup.  I assume we'll need the item to appear in another delta if we want to retry the attachment download, since we have no other way to target a single item for backup, meaning once skipped, we'll never be able to retrieve the attachment.  That actually sounds like a condition we'd want to list in `Known Issues`.  Have you double checked with Georgi and Vaibhav that we're okay with the permanently missing data here?  Or would it be better to record this situation as a recoverable error?

Reply:
> .. I assume we'll need the item to appear in another delta if we want to retry the attachment download ..
>  Have you double checked with Georgi and Vaibhav that we're okay with the permanently missing data here?

I had a conversation with Vaibhav and we finalized on skipping this for now. We want the backup to go through with this skipped. Similar to malware, it looks like this data will never be accessible from what we could find.

Reply:
> it looks like this data will never be accessible

Hmm, I didn't catch that there's a sense of permanence here, from the code comments.  Maybe you could add a small note about that somewhere in a comment, so it doesn't get confused for a temporary failure?